Libraries/TestHelpers.psm1

```diff
@@ -763,7 +763,7 @@ Function Get-SSHKey ($XMLSecretFile) {
 	if ($XMLSecretFile) {
 		$WebClient = New-Object System.Net.WebClient
 		$xmlSecret = [xml](Get-Content $XMLSecretFile)
-		$privateSSHKey = $xmlSecret.secrets.sshPrivateKey.InnerText
+		$privateSSHKey = $xmlSecret.secrets.sshPrivateKey
```

Comment:
thanks! I will fix it in a new PR, since we have sasurl format in our secrets file.

Reply:
So, do you not want me to make a change here?

Reply:
Once my PR merged, you can rebase.

Reply:
Removed this change.